### PR TITLE
Feature/account menu window

### PR DESCRIPTION
**Pull request by rshulyar**

### CODE REVIEW

**Comment by rshulyar** [comment]
https://rshulyar.atlassian.net/browse/GH-9

File: src/components/header/accountMenu/accountMenuWindow/accountMenuWindow.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+function AccountMenuWindow() {
+
+    return (
+        <div className="close-account-menu-window">
+            <div className="account-menu-window">
+                <AccountMenuWindowTop/> {/* pick a color */}
+                <AccountMenuWindowUpperMiddle/> {/* pick a color, create image */}
+                <AccountMenuWindowLowerMiddle/> {/* pick a color */}
+                <AccountMenuWindowBottom/> {/* pick a color */}
```
Comment (dobryanskyy) [inline src/components/header/accountMenu/accountMenuWindow/accountMenuWindow.js:16]: are these comments needed?

File: src/components/header/accountMenu/accountMenuWindowLowerMiddle/accountMenuWindowLowerMiddle.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    text-decoration-line: none;
+    text-decoration-style: solid;
+    text-decoration-thickness: auto;
+    user-select: text;
+    white-space: normal;
+    width: 41.6094px;
+}
+
+.a-amwlm:hover {
+    /* pick a color */
```
Comment (dobryanskyy) [inline src/components/header/accountMenu/accountMenuWindowLowerMiddle/accountMenuWindowLowerMiddle.css:91]: ?

File: src/components/header/accountMenu/accountMenuWindowUpperMiddle/accountMenuWindowUpperMiddle.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    font-variant-numeric: normal;
+    font-weight: 400;
+    height: 51px;
+    line-height: normal;
+    overflow-x: hidden;
+    user-select: text;
+    white-space: nowrap;
+    width: 354px;
+    border-top: 1px solid rgb(232, 234, 237);
+    /* border-bottom: 1px solid rgb(219, 220, 225); */
```
Comment (dobryanskyy) [inline src/components/header/accountMenu/accountMenuWindowUpperMiddle/accountMenuWindowUpperMiddle.css:20]: ??

**Review by dobryanskyy — CHANGES_REQUESTED** [review]

**Pull request merged**